api/src/main/java/marquez/db/Columns.java

```diff
@@ -78,6 +79,9 @@ private Columns() {}
   /* STREAM VERSION ROW COLUMNS */
   public static final String SCHEMA_LOCATION = "schema_location";
 
+  /* JOB ROW COLUMNS */
+  public static final String SYMLINK_TARGET_UUID = "symlink_target_uuid";
```

Comment:
Just a note, I can imagine the renaming of datasets also requiring a symlink target column.